### PR TITLE
Fix Windows compile

### PR DESCRIPTION
**Pull request by WolfDan**
Hey! 

Actually the library don't work on windows, but it is possible to make it compile on it, so I edited the mix file, added a Makefile.win and notes of how to build in windows

I ran the tests in my computer (Windows 10) and works just everything fine

Greetings

_PD: Is the first time I've done anything with Makefiles so feel free to edit anything that is wrong_

### CODE REVIEW

**Comment by ananthakumaran** [comment]
Thanks for the PR, I will try to set up CI in appveyor and then merge this into master.

File: README.md [hunk 1]
```diff
@@ -100,6 +100,16 @@ then you must have client binary `>= 5.1`. If you use `~>` in the
...
 
+## Additional Steps on Windows
+
+To compile the library in Windows you must have the Visual C++ Tools installed or VS 2017, if you don't use it probably you'll get a message telling you the `nmake` isn't installed
+
+- With Visual C++ Tools: Search for the file `vcvarsall.bat`, the Tools version 2017 are commonly located at `C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Auxiliary\Build` and run in a cmd the command `vcvarsall amd64`
+
+- With Visual Studio 2017 intalled: Go and type `Developer Command` in the search box and you will get the cmd programm as a result
+
+Then move to your folder directory and run `mix compile`
```
Comment (ananthakumaran) [inline README.md]: did you mean project directory?

**Comment by ananthakumaran** [comment]
windows CI builds are passing https://ci.appveyor.com/project/ananthakumaran/fdb/build/1.0.3 Will merge soon

File: README.md (hunk 1 above)
Comment (WolfDan) [reply README.md]: Oh yeah! my bad I'll fix it ^^

**Comment by ananthakumaran** [comment]
Just pushed a new release `5.1.7-5`. Let me know if you run into any problem with the new version in windows.

**Pull request merged**